packages/suite/src/utils/suite/analytics.ts

```diff
@@ -41,7 +41,8 @@ export const getSuiteReadyPayload = (state: AppState) => ({
     screenHeight: getScreenHeight(),
     platformLanguages: getPlatformLanguages().join(','),
     tor: getIsTorEnabled(state.suite.torStatus),
-    labeling: state.metadata.enabled ? state.metadata.provider?.type || 'missing-provider' : '',
+    // todo: duplicated with suite/src/utils/suite/logUtils
+    labeling: state.metadata.enabled ? state.metadata.providers.find(p => p.clientId === state.metadata.selectedProvider.labels)?.type || 'missing-provider' : '',
```

Comment:
please use `provider` instead of `p` when you will be unifying it with logsUtils

Reply:
`find` finds only first occurrence of `clientId`, use `filter` instead and than join the array using `,` Please add info to notion analytics, it is a breaking change, that it can include more providers. (From version bla bla it is this bla bla)

I added it to changelog
https://www.notion.so/satoshilabs/Changelog-Suite-1551ab666b1943f080ff56ffc6896d12

Change it also here with a comment that it was changed in version xxx
https://www.notion.so/satoshilabs/c63bdab01b704b099cab3bbb3227e1b1?v=406d107d6d584577a47d7580e8b2df89&p=8efc59a1a4b747bcb5155d04f355b7da&pm=s

Reply:
it is not possible to have multiple providers connected for one "metadata feature" see `metadata.selectedProviders`.